### PR TITLE
Update and rename template.json to B827EBFFFE728BB5.json

### PR DESCRIPTION
**Pull request by svgest**

### CODE REVIEW

**Comment by gonzalocasas** [comment]
@svgest please update with the requested changes, otherwise I'll close due to inactivity.

**Comment by svgest** [comment]
Hello,

 

I’m vgest, and I’m trying to find out what I have done wrong or what I have to do….

 

I’m trying to configure my RPi as a gateway to TTN, for that I have tried to edit the gateway-remote-config procedure.

But it seems that it is not that easy as one says it is.

 

What your part of this story is?  I don’t know…

 

Regards,

 

Staf

 

Van: sprokkie <notifications@github.com> 
Verzonden: dinsdag 18 december 2018 13:27
Aan: ttn-zh/gateway-remote-config <gateway-remote-config@noreply.github.com>
CC: svgest <info@excitron.be>; Mention <mention@noreply.github.com>
Onderwerp: Re: [ttn-zh/gateway-remote-config] Update and rename template.json to B827EBFFFE728BB5.json (#1139)

 

Hi,



Don’t understand ?

Who is svgest ? what hav I got to do with him ?



Please clatify for me.



Kind regards,

Ferry Meijlink



Van: Gonzalo Casas [mailto:notifications@github.com] 
Verzonden: dinsdag 18 december 2018 11:59
Aan: ttn-zh/gateway-remote-config <gateway-remote-config@noreply.github.com <mailto:gateway-remote-config@noreply.github.com> >
CC: sprokkie <sprokkie4@hotmail.com <mailto:sprokkie4@hotmail.com> >; Comment <comment@noreply.github.com <mailto:comment@noreply.github.com> >
Onderwerp: Re: [ttn-zh/gateway-remote-config] Update and rename template.json to B827EBFFFE728BB5.json (#1139)



@svgest <https://github.com/svgest> please update with the requested changes, otherwise I'll close due to inactivity.

File: B827EBFFFE728BB5.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "gateway_ID": "B827EBFFFE728BB5",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 51,362339530
```
[the remark this answers is not in the document]
Comment (svgest) [reply B827EBFFFE728BB5.json]: I'm was not aware that I did delete the template. So I'm very sorry.

**Comment by gonzalocasas** [comment]
@svgest thanks for the update! the file is now ok, the template seems to be still deleted, could you please try to re-add it in a new commit? Otherwise I could try to fix it later on, but if you do, I can merge asap.

Cheers!

**Comment by svgest** [comment]
Hello Gonzalo,

 

I’m a little bit confused.

When I go to the gateway-remote-config, I can see the template.json file at the end of the list and can look into the file.

 

Or is there something wrong with my json-file ?

 

 

 

 

 

Van: Gonzalo Casas <notifications@github.com> 
Verzonden: dinsdag 18 december 2018 14:48
Aan: ttn-zh/gateway-remote-config <gateway-remote-config@noreply.github.com>
CC: svgest <info@excitron.be>; Mention <mention@noreply.github.com>
Onderwerp: Re: [ttn-zh/gateway-remote-config] Update and rename template.json to B827EBFFFE728BB5.json (#1139)

 

@svgest <https://github.com/svgest>  thanks for the update! the file is now ok, the template seems to be still deleted, could you please try to re-add it in a new commit? Otherwise I could try to fix it later on, but if you do, I can merge asap.

Cheers!

**Comment by gonzalocasas** [comment]
@svgest yay! cool stuff! merged! thanks!!

**Pull request merged**